sqlglot/dialects/oracle.py

```diff
@@ -196,6 +196,8 @@ class Generator(generator.Generator):
             exp.ToChar: lambda self, e: self.function_fallback_sql(e),
             exp.Trim: trim_sql,
             exp.UnixToTime: lambda self, e: f"TO_DATE('1970-01-01','YYYY-MM-DD') + ({self.sql(e, 'this')} / 86400)",
+            exp.VariancePop: rename_func("VAR_POP"),
+            exp.Variance: rename_func("VAR_SAMP"),
```

Comment:
Even though `VARIANCE` is usually an alias to `VAR_SAMP`, it seems that they're not completely interchangeable in [Oracle](https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/VAR_SAMP.html#GUID-314D5831-0E26-4ABF-9F46-35F78F97DA52):

> This function is similar to VARIANCE, except that given an input set of one element, VARIANCE returns 0 and VAR_SAMP returns null.

Perhaps we should capture this nuance with an optional arg in `Variance`?

Reply:
Woof.  Ok, so an optional arg to allow a user to explicitly request translating to `VARIANCE` instead of `VAR_SAMP`?